cli/cycpp.py

```diff
@@ -1568,10 +1578,10 @@ def impl(self, ind="  "):
 
             if key in BUFFERS:  # buffer state, skip
                 continue
-            if 'derived_init' in info:  # derived state, skip
+            if 'internal' in info:  # derived state, skip
                 continue
 
-            opt = True if 'default' in info else False
+            opt = True if 'default' in info or 'derived_init' in info else False
```

Comment:
You've convinced me it is not as bad as I initially thought.  I'll change.
I don't have time to do the tests though.  I need to be moving on to other
things.

On Wed, Apr 1, 2015 at 3:24 PM, Matthew Gidden <notifications@github.com>
wrote:

> In cli/cycpp.py
> <https://github.com/cyclus/cyclus/pull/1110#discussion_r27613345>:
>
> >                  continue
> >
> > -            opt = True if 'default' in info else False
> > +            opt = True if 'default' in info or 'derived_init' in info else False
>
> In other words, the way derived_init works with this PR should be
> different than it works now. Cycamore facilities should have to add an
> 'internal' key to get the same behavior.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/cyclus/pull/1110/files#r27613345>.
>